features/networking/multus.feature

```diff
@@ -170,3 +170,97 @@ Feature: Multus-CNI related scenarios
     #  | curl | --connect-timeout | 5 | <%= cb.pod1_multus_ip %>:8080 |
     #Then the step should succeed
     #And the output should contain "Hello OpenShift"
+
+  # @author bmeng@redhat.com
+  # @case_id OCP-21853
+  @admin
+  Scenario: Create pods with multus-cni - host-device
+    # Make sure that the multus is enabled
+    Given the master version >= "4.0"
+    And the multus is enabled on the cluster
+    And I select a random node's host
+    And evaluation of `node.name` is stored in the :target_node clipboard
+
+    # Add a host-device which will be used by the pod later
+    Given I run the :get admin command with:
+      | resource      | pod                                 |
+      | n             | openshift-sdn                       |
+      | fieldSelector | spec.nodeName=<%= cb.target_node %> |
+      | l             | app=sdn                             |
+      | o             | jsonpath={.items[0].metadata.name}  |
+    Then the step should succeed
+    And evaluation of `@result[:response]` is stored in the :target_sdnpod clipboard
+
+    Given I run the :exec admin command with:
+      | pod              | <%= cb.target_sdnpod %>                              |
+      | n                | openshift-sdn                                        |
+      | oc_opts_end      |                                                      |
+      | exec_command     | sh                                                   |
+      | exec_command_arg | -c                                                   |
+      | exec_command_arg | ip link add eth08 link eth0 type macvlan mode bridge |
```

Comment:
hardcoded device name may prevent concurrent execution. Can we use a random device name instead of hardcoded `eth08`?

Reply:
I have updated the interface name in the other PR. https://github.com/openshift/verification-tests/pull/161

Thanks.